packages/hunter/package.json

```diff
@@ -15,6 +15,7 @@
     "@alercebroker/http-client": "^1.1.0-rc.16",
     "@quasar/extras": "^1.15.0",
     "@vue/test-utils": "^2.0.2",
+    "axios": "^0.27.2",
```

Comment:
I guess this shouldn't be a direct dependency here ?

Reply:
nope, I installed it just for testing purposes